qt/widgets/common/inc/MantidQtWidgets/Common/HintStrategy.h

```diff
@@ -22,11 +22,14 @@ class EXPORT_OPT_MANTIDQT_COMMON HintStrategy {
   HintStrategy(){};
   virtual ~HintStrategy() = default;
 
-  /** Create a list of hints for auto completion
+  /** Create a list of hints for auto completion.
+    * This is overwritten on the python side.
+    * However, leaving this as abstract method causes SIP issues
+    * in _common.sip
 
       @returns A map of keywords to short descriptions for the keyword.
    */
-  virtual std::vector<Hint> createHints() = 0;
```

Comment:
Could you help me understand the need to make this not pure?

Reply:
This was a change I forgot to take out from when the SIP wouldn't compile properly. I had to change the way we add `createHints` on the python side and meant this couldn't be pure.

I'll change this back

Reply:
:+1: